src/models/data.h

```diff
@@ -58,6 +60,10 @@ struct Symbol
     QString symbol;
     // prettified function name
     QString prettySymbol;
+    // relative address
+    quint64 relAddr;
```

Comment:
= 0, also below

Reply:
Sorry, what do you mean? I set default value 0 for it.